src/actions/cloud-compute/cloud-compute.ts

```diff
@@ -126,106 +159,86 @@ export const runMeticulousTestsCloudComputeAction = async (): Promise<void> => {
     logger.info(`Generating visual snapshots for commit ${shortSha(head)}`);
   }
 
-  try {
-    await throwIfCannotConnectToOrigin(appUrl);
-
-    const onTunnelCreated = ({
-      url,
-      basicAuthUser,
-      basicAuthPassword,
-    }: {
-      url: string;
-      basicAuthUser: string;
-      basicAuthPassword: string;
-    }) => {
-      logger.info(
-        `Secure tunnel to ${appUrl} created: ${url}, user: ${basicAuthUser}, password: ${basicAuthPassword}`
-      );
+  await throwIfCannotConnectToOrigin(appUrl);
+
+  const onTunnelCreated = ({
+    url,
+    basicAuthUser,
+    basicAuthPassword,
+  }: {
+    url: string;
+    basicAuthUser: string;
+    basicAuthPassword: string;
+  }) => {
+    logger.info(
+      `Secure tunnel to ${appUrl} created: ${url}, user: ${basicAuthUser}, password: ${basicAuthPassword}`
+    );
 
-      if (isDebugPRRun) {
-        updateStatusComment({
-          octokit,
-          event,
-          owner,
-          repo,
-          body: `🤖 Meticulous is running in debug mode. Secure tunnel to ${appUrl} created: ${url} user: \`${basicAuthUser}\` password: \`${basicAuthPassword}\`.\n\n
+    if (isDebugPRRun) {
+      updateStatusComment({
+        octokit,
+        event,
+        owner,
+        repo,
+        body: `🤖 Meticulous is running in debug mode. Secure tunnel to ${appUrl} created: ${url} user: \`${basicAuthUser}\` password: \`${basicAuthPassword}\`.\n\n
 Tunnel will be live for up to ${DEBUG_MODE_KEEP_TUNNEL_OPEN_DURAION.toHuman()}. Cancel the workflow run to close the tunnel early.`,
-          testSuiteId: "__meticulous_debug__",
-          shortHeadSha: shortCommitSha(head),
-          createIfDoesNotExist: true,
-        }).catch((err) => {
-          logger.error(err);
-        });
-      }
-    };
-
-    const keepTunnelOpenPromise = isDebugPRRun ? defer<void>() : null;
-    let keepTunnelOpenTimeout: NodeJS.Timeout | null = null;
-
-    let lastSeenNumberOfCompletedTestCases = 0;
-
-    const onProgressUpdate = (testRun: TestRun) => {
-      if (
-        !IN_PROGRESS_TEST_RUN_STATUS.includes(testRun.status) &&
-        keepTunnelOpenPromise &&
-        !keepTunnelOpenTimeout
-      ) {
-        logger.info(
-          `Test run execution completed. Keeping tunnel open for ${DEBUG_MODE_KEEP_TUNNEL_OPEN_DURAION.toHuman()}`
-        );
-        keepTunnelOpenTimeout = setTimeout(() => {
-          keepTunnelOpenPromise.resolve();
-        }, DEBUG_MODE_KEEP_TUNNEL_OPEN_DURAION.as("milliseconds"));
-      }
+        testSuiteId: "__meticulous_debug__",
```

Comment:
we should probably tag this comment with the project ID too as this won't work in multi-project setups. Looks like this already doesn't work well on multi-project PRs.